frontend/app/controllers/projects/create-new.js

```diff
@@ -0,0 +1,164 @@
+  SelectVersionValidations = SelectVersionValidations;
+  // Services
+  @service store;
+  @service notification;
+
+  @tracked modalForm = true;
+  @tracked project = getEmptyChangeset(
+    ProjectValidations,
+    this.store.createRecord('project')
+  );
+
+  @tracked addVersionChangeset = getEmptyChangeset(
+    DependencyVersionValidations
+  );
+  @tracked dependencyChangeset = getEmptyChangeset(SelectDependencyValidations);
+  @tracked versionChangeset = getEmptyChangeset(SelectVersionValidations);
+  @tracked addDependencyChangeset = getEmptyChangeset(DependencyValidations);
+
+  @tracked selectedDependency = null;
+
+  modalSequence = tracked({
+    selectDependency: false,
+    selectVersion: false,
+    addDependency: false,
+    addVersion: false,
+  });
+
+  get modals() {
+    const maxNum = Math.max(...Object.values(this.modalSequence));
+    return Object.entries(this.modalSequence).reduce((acc, [key, value]) => {
+      acc[key] = value === maxNum && value !== 0;
+      return acc;
+    }, {});
+  }
+  get filteredDependencyVersions() {
+    const dep = this.selectedDependency;
+    if (!dep) return [];
+    return this.store.peekAll('dependencyVersion').filter((dv) => {
+      return dv.dependency.get('id') === dep.id;
+    });
+  }
+
+  @action openModal(modal) {
+    const maxNum = Math.max(...Object.values(this.modalSequence));
+
+    if (typeof this.modalSequence[modal] !== 'number') {
+      this.modalSequence[modal] = maxNum + 1;
+      return;
+    }
+
+    const currentnNum = this.modalSequence[modal];
+
+    if (currentnNum === maxNum) {
+      return;
+    }
+    Object.entries(this.modalSequence).forEach(([key, value]) => {
+      if (value > currentnNum) {
+        this.modalSequence[key] = value - 1;
+      }
+    });
+    this.modalSequence[modal] = maxNum;
+  }
+
+  @action closeModal(modal) {
+    if (typeof this.modalSequence[modal] !== 'number') {
+      return;
+    }
+    const currentnNum = this.modalSequence[modal];
+    for (const [key, value] of Object.entries(this.modalSequence)) {
+      if (value > currentnNum) {
+        this.modalSequence[key] = value - 1;
+      }
+    }
+    this.modalSequence[modal] = false;
+  }
+
+  @action closeModals() {
+    Object.keys(this.modalSequence).forEach((key) => {
+      this.modalSequence[key] = false;
+    });
+    this.dependencyChangeset.rollback();
+    this.versionChangeset.rollback();
+    this.addVersionChangeset.rollback();
+    this.addDependencyChangeset.rollback();
+    this.selectedDependency = null;
+  }
+
+  @action
+  async addVersion(dependencyVersion) {
+    try {
+      const depver = await this.store.createRecord(
+        'dependencyVersion',
+        dependencyVersion.pendingData
+      );
+      depver.dependency = this.selectedDependency;
+      await depver.save();
+      this.project.dependencyVersions.pushObject(depver);
+      this.closeModals();
+    } catch (e) {
+      this.notification.danger(e), { pos: 'bottom-right' };
```

Comment:
```suggestion
      this.notification.danger(e, { pos: 'bottom-right' });
```

Reply:
If you are using `pos: "bottom-right"` everywhere, consider configuring it globally:

```js
// config/environment.js

module.exports = function (/* environment */) {
  const ENV = {
    "ember-uikit": {
      notification: {
        pos: "bottom-right",
      },
    },
  };

  return ENV;
};
```

Reply:
resolved as of https://github.com/adfinis/Outdated/pull/14/commits/805c733a5f1cbcef9d419eefc5f57650db2bd6b9